specification/search/resource-manager/Microsoft.Search/stable/2015-08-19/search.json

```diff
@@ -1064,6 +1064,7 @@
           "type": "string",
           "description": "The identity type.",
           "enum": [
+            "None",
```

Comment:
@natinimni Adding a new enum value is a client-side breaking change, so make sure to bump the major version of the .NET Management SDK.

Reply:
It also wouldn't hurt to ensure there's an example of using this value.

Reply:
thanks @brjohnstmsft ! 
The .NET Mgmt SDK was never generated with the identity property, so it should not be a breaking change for it :) 
I've added some examples.

Reply:
Good to know. Thanks!